portal/src/main/webapp/js/SpecRunner.html

```diff
@@ -1,99 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
```

Comment:
No. This is an independent page showing some old test results based on jasmine but even those tests are not working anymore. Here is the screenshot. 
![image](https://cloud.githubusercontent.com/assets/5973438/19781558/be6196b8-9c57-11e6-8e25-a5a45a6d7595.png)